.circleci/config.yml

```diff
@@ -120,9 +120,11 @@ commands:
       - run: |
           tool/test/start-core-server.sh
           python3.8 -m pip install wheel 
-          python3.8 -m pip install --extra-index-url https://repo.vaticle.com/repository/pypi-snapshot/simple typedb-driver==0.0.0+$(git rev-parse HEAD)
+          python3.8 -m pip install --extra-index-url https://repo.typedb.com/public/public-snapshot/python/simple typedb-driver==0.0.0+$(git rev-parse HEAD)
           sleep 3
-          (cd python/tests/deployment/ && python3.8 -m unittest test && export TEST_SUCCESS=0 || export TEST_SUCCESS=1)
```

Comment:
We previously removed `(cd ... && export )` was run in a subprocess so `TEST_SUCCESS` was not exported.
The C and C++ tests use the `pushd ... popd` style